tests/AccountManagerTest.java

```diff
@@ -0,0 +1,48 @@
+import org.junit.*;
+
+import java.io.File;
+import java.net.URL;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+
+/**
+ * This class consists of JUnit tests to test the logic and functionality of the Account Manager class methods and
+ * ensure the methods are functioning as required.
+ *
+ * @author chad_olsen
+ * @since 2017/06/02.
+ */
+public class AccountManagerTest {
+    private static AccountManager manager;
+    private static Map<String, Account> registeredAccounts;
+    private static Account newAccount;
+
+    @BeforeClass
+    public static void start(){
+        manager = new AccountManager();
+        registeredAccounts = new LinkedHashMap<>();
+        newAccount = new Account("test", "test", new Catalogue());
+        registeredAccounts.put("test", newAccount);
+    }
+
+    @Test
+    public void writeToAccount() throws Exception {
+        URL url = getClass().getResource("AccountManager.dat");
+        File file = new File(url.getPath());
+        assertTrue(file.exists());
```

Comment:
Shouldn't this assert false? Shouldn't the file not exist until an account has been written?